gw_eccentricity/eccDefinition.py

```diff
@@ -274,6 +306,13 @@ def __init__(self, dataDict, num_orbits_to_exclude_before_merger=2,
                 USE THIS WITH CAUTION!
         """
         self.precessing = precessing
+        self.frame = frame
+        # check if frame makes sense.
+        self.available_frames = ["inertial", "coprecessing"]
```

Comment:
Does `available_frames` needs to be in self? If not, don't put it in self, just makes self bloated, and makes it harder to find things you really need in self if you try to tab complete in a notebook.

Reply:
removed it from `self`